### PR TITLE
Rename aro config to indicate upi installer being used

### PR DESCRIPTION
**Pull request by svetsa-rh**

### CODE REVIEW

**Comment by svetsa-rh** [comment]
cc: @krishvoor

**Comment by krishvoor** [comment]
@svetsa-rh can you address the failing updates.. otherwise we are good to go.

**Comment by paigerube14** [comment]
Are we able to add 'upi' to the workflow name as well? I think we, by default, think that it would be ipi installation

**Comment by svetsa-rh** [comment]
> Are we able to add 'upi' to the workflow name as well? I think we, by default, think that it would be ipi installation

@paigerube14 Good catch. I updated it.

File: ci-operator/step-registry/openshift-qe/installer/azure/aro/upi/openshift-qe-installer-azure-aro-upi-workflow.metadata.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"path": "openshift-qe/installer/azure/aro/upi/openshift-qe-installer-azure-aro-upi-workflow.yaml",
+	"owners": {}
```
[the remark this answers is not in the document]
Comment (svetsa-rh) [reply ci-operator/step-registry/openshift-qe/installer/azure/aro/upi/openshift-qe-installer-azure-aro-upi-workflow.metadata.json:3]: Will delete that file and try to recreate.

[the remark this answers is not in the document]
Comment (svetsa-rh) [reply ci-operator/step-registry/openshift-qe/installer/azure/aro/upi/openshift-qe-installer-azure-aro-upi-workflow.metadata.json]: Done! Looks fixed now.

**Pull request merged**